Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1015,17 +1015,23 @@ public static void HandleCI(Session session, params string[] parameters)
             WorldObject testContainer = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
```

Comment:
See my comment below.   That has a bug when you try to persist as far as I can tell.   It is also a bit of in indirect to go from aceObject -> worldobject - clone for the reasons listed below.   I think it is better to just use the aceobject of the weenie clone and reset for a new object then worldobject from that new ace object.   Either will work if you fix the issue saving exposed - I think the way I am doing it is cleaner.